### PR TITLE
Added test with new endpoint

### PR DESCRIPTION
**Pull request by SergioBertolinSG**
There is a problem here, same test using new endpoint is not working.

### CODE REVIEW

File: build/integration/features/webdav-related.feature [hunk 1]
```diff
@@ -9,6 +9,13 @@ Feature: webdav-related
 		When User "user0" moves file "/textfile0.txt" to "/FOLDER/textfile0.txt"
 		Then the HTTP status code should be "201"
 
+	Scenario: moving a file new endpoint way
```
Comment (DeepDiver1975) [inline build/integration/features/webdav-related.feature]: May I propose to start a new file for v2 webdav endpoint? THX

Comment (SergioBertolinSG) [inline build/integration/features/webdav-related.feature]: Sure. The main problem here is that the same behaviour is not working with the new endpoint, it should work isn't?

Comment (DeepDiver1975) [inline build/integration/features/webdav-related.feature]: Yes - all tests on v1 should also work on v2 - if you experience diffs we need to have a closer look. THX

**Comment by PVince81** [comment]
@SergioBertolinSG any chance to get this in mergable state, even if it doesn't cover all cases yet ?

**Comment by SergioBertolinSG** [comment]
Updated this branch. There is a problem with the new endpoint, files cannot be find:

Client error response [url] http://localhost:8080/remote.php/dav/files/user0/textfile0.txt [status code] 404 [reason phrase] Not Found (GuzzleHttp\Exception\ClientException)

Using cadaver happens the same, this requires a fix.

**Comment by SergioBertolinSG** [comment]
Raised here the bug https://github.com/owncloud/core/issues/24776

**Comment by PVince81** [comment]
The bug was fixed. Please rebase to make the tests pass.

**Comment by SergioBertolinSG** [comment]
@PVince81 rebased and fixed an error.

File: build/integration/features/dav-v2.feature [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		And As an "user0"
+		When Downloading file "/files/user0/welcome.txt" with range "bytes=51-77"
+		Then Downloaded content should be "example file for developers"
+
+	Scenario: Downloading a file on the new endpoint should serve security headers
+		Given using dav path "remote.php/dav/files/admin/"
+		And As an "admin"
+		When Downloading file "welcome.txt"
+		Then The following headers should be set
+			|Content-Disposition|attachment|
```
Comment (PVince81) [inline build/integration/features/dav-v2.feature]: this will fail if you rebase as we changed the content-disposition

**Comment by PVince81** [comment]
@SergioBertolinSG please rebase, there were changes with the Content-Disposition header on master

**Comment by PVince81** [comment]
Otherwise it looks good.

**Comment by SergioBertolinSG** [comment]
@DeepDiver1975 there has been changes in how the job ocs-api-integration-tests-ci is run?
All the tests passes locally to me. (Using mysql)

**Comment by PVince81** [comment]
Rebased again for CI.

Let's hope the Sabre version for composer will not cause any more timeouts here...

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**